oommfc/scripts/driver.py

```diff
@@ -109,9 +109,9 @@ def driver_script(
                     "be specified explicitely."
                 )
                 raise RuntimeError(msg)
-            elif mm.ZhangLi() in system.dynamics:
+            elif system.dynamics.get(type=mm.ZhangLi):
```

Comment:
If you decide to go for `Container` as the return type of `get`, then `Container` would need `__bool__` to allow this logic.
```python
def __bool__(self):
    return bool(self._terms)
```

Reply:
Aha, I did not know we have `__len__` for container :)